core/spec/models/spree/adjustment_spec.rb

```diff
@@ -13,6 +13,32 @@
 
   let(:adjustment) { Spree::Adjustment.create!(label: 'Adjustment', adjustable: order, order: order, amount: 5) }
 
+  describe 'scopes' do
```

Comment:
@Mafi88 - Use of context block doesn't seems good to me here. if you still feels we should add one, please let me know.